docs/reference/commandline/network_create.md

```diff
@@ -188,6 +190,33 @@ $ docker network create \
     simple-network
 ```
 
+### Transparent driver options
+The `transparent` network on Windows is very similar to the Linux `bridge` driver.
```

Comment:
Can you add a blank line between the heading and the body-text? (I think some Markdown renderers don't like it if it's missing)